python/scisql/configure.py

```diff
@@ -92,7 +91,7 @@ def apply_tpl(src_file, target_file):
     out_cfg = t.safe_substitute(**params_dict)
     for match in t.pattern.findall(t.template):
```

Comment:
As far as I can tell, though [the docs](https://docs.python.org/2/library/string.html#string.Template) would seem to agree with you, it looks like the metaclass for `string.Template` turns it into a pattern object for you (see `_TemplateMetaclass` in `lib/python2.7/string.py`). @fjammes - do you agree?